business/dashboards.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/kiali/kiali/models"
 	"github.com/kiali/kiali/prometheus"
 	"github.com/kiali/kiali/prometheus/internalmetrics"
+	"k8s.io/client-go/tools/clientcmd/api"
```

Comment:
following the kiali coding standards, third party libs go in a second group of imports between the standard go libs and the kiali libs with blank lines separating. So move this so it looks something like this:

```
...
	"sync"

	"k8s.io/client-go/tools/clientcmd/api"

	"github.com/kiali/kiali/config"
...
```